CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ for more information.
   - Added report section listing samples that had no adapters trimmed
 - **RSeQC**
   - Fix `ZeroDivisionError` error for `bam_stat` results when there are 0 reads ([#1735](https://github.com/ewels/MultiQC/issues/1735))
+  - Fix "max() arg is an empty sequence" error ([#1985](https://github.com/ewels/MultiQC/issues/1985))
```

Comment:
Needs moving up to the v1.16dev changelog section (I just updated your branch, which was a bit behind).